boards/common/st/configs/stm32l0.cfg

```diff
@@ -1,3 +1,3 @@
 source [find target/stm32l0.cfg]
-reset_config srst_only
+reset_config srst_only connect_assert_srst
```

Comment:
Why do you change the `reset_config` configurations in a _"boards/common/stm32-link: add common place for stlink"_ commit ?
If it it necessary, it should be another properly justified commit.